CMakeLists.txt

```diff
@@ -268,24 +268,23 @@ if(SENTRY_WITH_LIBUNWINDSTACK)
 endif()
 
 if(SENTRY_BACKEND_CRASHPAD)
-	add_subdirectory(external/crashpad EXCLUDE_FROM_ALL)
-	target_include_directories(sentry PRIVATE
-		"$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/external/crashpad>"
-		"$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/external/crashpad/third_party/mini_chromium/mini_chromium>"
-	)
-	# we include a crashpad header, which itself needs C++14 support
-	target_compile_features(sentry PUBLIC cxx_std_14)
-	target_link_libraries(sentry PRIVATE crashpad_client crashpad_util)
-	if(NOT BUILD_SHARED_LIBS)
-		sentry_install(TARGETS crashpad_client crashpad_util crashpad_compat getopt mini_chromium zlib EXPORT sentry
-			LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
-			ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
-		)
+	option(SENTRY_CRASHPAD_SYSTEM "Use system crashpad" OFF)
```

Comment:
nice! Our specific crashpad fork has support for attachments, without which the integration is pretty useless. But its good to have this optional support here.